src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+type State = {
+  today: Date;
+};
+
+type Props = {
+  clockName: string;
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  clockId = 0;
+
+  componentDidMount() {
+    this.clockId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      this.printCurrentTime();
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (this.props.clockName !== prevProps.clockName) {
+      window.console.debug(
+        `Renamed from ${prevProps.clockName} to ${this.props.clockName}`,
+      );
+    }
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.clockId);
+  }
+
+  printCurrentTime() {
+    window.console.info(
+      this.state.today.toUTCString().slice(-12, -4),
+    );
+  }
```

Comment:
```suggestion
  getFormattedTime(value: ...): ... {
    return ...
  }
```
Make it useful for all cases. You can reuse it twice in your code.